amqphandler/amqphandler.go

```diff
@@ -537,12 +537,12 @@ func (handler *AmqpHandler) setupReceiveConnection(scheme string,
 
 	handler.wg.Add(1)
```

Comment:
Let's disable unparam lint and drop this commit?

Reply:
done